carbon-design-system/carbon
# fix(a11y): fixing MultiSelect no focus indicator visible on clear button

Closes #6855

Fixing MultiSelect - accessibility issue - no focus indicator visible on clear button

#### Changelog

**Changed**

MultiSelect component

#### Testing / Reviewing

- Go to http://localhost:9000/?path=/story/multiselect--default
- Select a few elements
- Focus on the button to "Clear Selection" by keyboard, it should show an outline
